src/services/pcn-packetcapture/src/Packetcapture_dp_egress.c

```diff
@@ -193,6 +198,13 @@ static __always_inline int handle_rx(struct CTXTYPE *ctx, struct pkt_metadata *m
 
   }
 
+  /* Getting packet timestamp */
```

Comment:
Add a comment such as "The algorithm for timestamping packets is described in file XX function YY"